scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -688,17 +688,22 @@ static __always_inline void async_p2dq_enqueue(struct enqueue_promise *ret,
 			return;
 		}
 
-		taskc->dsq_id = cpu_dsq_id(taskc->dsq_index, cpuc);
 		update_vtime(p, cpuc, taskc, llcx->vtime);
+		if (is_idle) {
+			scx_bpf_dsq_insert(p, SCX_DSQ_LOCAL_ON|cpu, taskc->slice_ns, enq_flags);
+			stat_inc(P2DQ_STAT_IDLE);
+			scx_bpf_kick_cpu(cpu, SCX_KICK_IDLE);
```

Comment:
Local DSQ insert shouldn’t need the idle kicking, right? Otherwise LGTM.

Reply:
I think the extra kick may actually regress performance a bit, the core is already triggering a `resched_curr()` when it's needed with local DSQs, the explicit kick may trigger another unnecessary `resched_curr()` + there's the (tiny) overhead of the kfunc call, but the potential of an additional  `resched_curr()` is probably more impactful.